Docs update for deploying with Kustomize (#47772)
sourcegraph/sourcegraph: doc/admin/deploy/index.md

    </div>
  </a>
</div>
  
### [Machine Images](machine-images/index.md) 

**For Enterprises looking for a self-hosted solution on Cloud.** 

<span class="badge badge-note">RECOMMENDED</span> An option to run Sourcegraph on your own infrastructure using pre-configured machine images.

Customized machine images allow you to spin up a preconfigured and customized Sourcegraph instance with just a few clicks, all in less than 10 minutes! Currently available in the following hosts:

<div class="getting-started">
  <a class="btn btn-secondary text-center" href="machine-images/aws-ami"><span>AWS AMIs</span></a>
  <a class="btn btn-secondary text-center" href="machine-images/azure"><span>Azure Images</span></a>
  <a class="btn btn-secondary text-center" href="machine-images/gce"><span>Google Compute Images</span></a>
</div>

### [Install-script](single-node/script.md)

**For Enterprises looking for a generic self-hosted solution.** 

The [install-script](single-node/script.md) is recommended for on-premises deployments to unsupported cloud environments. You can also use it to set up Linux VMs for your own machine images.

> >NOTE: Deploying with machine images require the necessary technical expertise and resources to maintain and manage their own infrastructure.

Reviewer: ```suggestion
>NOTE: Deploying with machine images requires technical expertise and the ability to maintain and manage your own infrastructure.
```

> >NOTE: Deploying with machine images require the necessary technical expertise and resources to maintain and manage their own infrastructure.

Author: Updated by warren in one of his commit

### [Kubernetes](kubernetes/index.md)

**For large Enterprises that require a multi-node, self-hosted solution.**

> - **Kustomize** utilizes the built-in features of kubectl to provide maximum flexibility in configuring your deployment

Reviewer: Should we mention the relationship of this to ease of upgrades?

Author: They are both easy to upgrade when follow correctly 😆
